src/test/java/uk/ac/cam/cl/dtg/segue/database/GitDbTest.java

```diff
@@ -54,7 +54,7 @@ public void getTreeWalk_checkThatBlankPathsAreAllowed_noExceptionThrown() throws
     replay(git);
     replay(repo);
 
-    assertNull(db.getTreeWalk("sha",
-        "")); // Blank path is explicitly allowed. This should not throw an exception. But in this case we've passed an invalid sha, so we should get null back.
+    assertNull(db.getTreeWalk("sha", "")); // Blank path is explicitly allowed.
```

Comment:
I know you're just copying what was already here, but we shouldn't write a comment when instead we could just make it part of the error message.